Automattic/wp-calypso
# DIFM: add refund text on the page picker step

Fixes https://github.com/Automattic/martech/issues/3097

## Proposed Changes

* Adds the text "14-day money back guarantee" to the page picker step of the DIFM flow.

## Why are these changes being made?


* The text provides essential information about the refund policy at a critical step in the process. Since the purchase amount for this product is substantial, it also would make users more comfortable about making the purchase.

## Testing Instructions



* Go to `/start/do-it-for-me` and go through the flow steps till you reach the page picker step.
* Confirm that you see the "14 day money back guarantee" text below the total.
* <img width="537" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/7f92f7c5-d822-4872-a9b6-f279b6199df8">

#### Regression Testing
* Since the icon is also used in another place, make sure that it is correctly displayed by going to `/start/plans?flags=onboarding/emphasize-refund-period-in-plans-step` and confirm that the top banner is shown.
* <img width="1906" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/f00b66fe-ed3d-4967-be43-48a42ce14de9">


## Pre-merge Checklist
